### PR TITLE
Update user map for March, April 2017 #7215

### LINKED ISSUE #7215: Update user map for March, April 2017
Refer #6825 

Last updated date should be April 30 2017

Data:
```
Australia	Australian Institute of Business
Australia	Charles Sturt University
Australia	Holy Eucharist School
Australia	King's Own Institute
Australia	Latrobe High School
Australia	Macquarie University
Australia	Swinburne University of Technology
Australia	University of Sydney
Australia	Western Sydney University
Belgium	Artevelde University
Belgium	Howest
Belgium	KA Avelgem
Belgium	Solvay Brussels School
Belgium	Vives University College
Belgium	Vrije Universiteit Brussel
Bermuda	Bermuda Institute
Brazil	AEDB
Brazil	Unicesumar
Brazil	Universidade Católica de Brasília
Brazil	Universidade de Sao Paulo
Canada	Algonquin College
Canada	Balmoral Drive Senior Public School
Canada	Burnaby North Secondary School
Canada	Cegep de Bois-de-Boulogne
Canada	Chetwynd Secondary School
Canada	College Ahuntsic
Canada	Dalhousie University 
Canada	Eton College
Canada	Georgian College
Canada	Hamilton District Christian High
Canada	Queen's University
Canada	Royal Roads University
Canada	Seneca College
Canada	Simon Fraser University
Canada	Toronto District School Board
Canada	University of British Columbia
Canada	University of Winnipeg
Denmark	Birkerød Gymnasium
Denmark	VIA University College
Egypt	German University in Cairo
Finland	Centria University of Applied Sciences
FInland	Lappeenranta University of Technology
France	Emlyon Business School
France	Hec Paris
France	IESEG School of Management
France	SKEMA Business School
Germany	Bielefeld University 
Germany	SHIFTSCHOOL
Germany	Technical University Munich
Greece	Athens University of Economics and Business
Greece	University of Sheffield
Hong Kong	CCC Kei Yuen College
Hong Kong	HKUST
Hungary	Codecool
India	Ahmedabad University
India	IIIT Delhi
India	IIT Bhubaneswar
India	Jadavpur University
India	MakerWala
India	NIT Hamirpur
Indonesia	Institut Teknologi Bandung
Indonesia	School of Business and Management
Japan	AdRoll
Malaysia	Universiti Malaysia Perlis
Malaysia	Universiti Malaysia Terengganu
Malaysia	Universiti Utara Malaysia
Malaysia	UTM 
Mauritius	African Leadership University
Mexico	Latin American School
Mexico	Universidad Tecmilenio
Mexico	Unosquare
Netherlands	Eindhoven University of Technology
Netherlands	Gymnasium Juvenaat
Netherlands	Heutink ICT
Netherlands	HPE
Netherlands	Saxion University of Applied Sciences
Netherlands	Technical University of Eindhoven
New Zealand	University of Auckland
Norway	Norwegian School of Economics
Pakistan	National University of Computer And Emerging Sciences
Philippines	University of St. La Salle
Scotland	Mearns Castle High School
Singapore	Civil Service College
Singapore	Halogen Foundation Singapore
Singapore	Hwa Chong Institution
Singapore	Nanyang Technological University
Singapore	Punggol Secondary School
Singapore	Singapore Institute of Technology
Singapore	Singapore Polytechnic
South Korea	KyungHee University
Spain	Centre Universitari de Disseny
Sri Lanka	University of Kelaniya
Sweden	Jönköping University
Sweden	Linnaeus University
Taiwan	Yuan Ze University
Tanzania	University of Dar es Salaam
Thailand	Yala Medical Education Center
Turkey	Hacettepe Üniversitesi
Turkey	Middle East Technical University
UK	London School of Economics and Political Science
UK	Robert Gordon University
UK	Robert Gordon University
UK	University of Bradford
UK	University Of Bradford
UK	University of East London
UK	University of St Andrews
UK	University of the West of England
UK	University of Westminster
USA	Abilene Christian University
USA	Baylor University
USA	Baylor University
USA	Belmont Unviersity 
USA	Boston University
USA	Brigham Young University-Idaho
USA	Cabrini University
USA	Cal-Poly Pomona University
USA	Carnegie Mellon University
USA	Central Michigan University
USA	Central Washington University
USA	Chapman University
USA	Christian Brothers High School
USA	City University of New York
USA	Community Partnership Charter School
USA	CONNSTEP
USA	Cornell University
USA	Cornell University
USA	Cuyahoga Community College
USA	Drexel University
USA	Duke University
USA	Duluth Middle School
USA	East Carolina University
USA	Eastern Michigan University 
USA	El Paso Community College
USA	Florida Atlantic University
USA	Florida Gulf Coast University
USA	Florida International University
USA	Francis Lewis High School
USA	Gannon University 
USA	George Washington University
USA	Georgia Gwinnett College
USA	Georgia Tech
USA	Grossmont Community College
USA	Harding University
USA	Harrisburg University of Science & Technology
USA	Henrico County Schools
USA	Heritage Baptist Academy
USA	Indiana University
USA	Iowa City West High School
USA	Iowa State University
USA	Johnson & Wales University
USA	Kansas State University
USA	Liberty High School
USA	Liberty University
USA	Lone Star College
USA	Long Island University
USA	Longwood University
USA	Louisiana State University
USA	Loveland High School
USA	Manhattan College
USA	Metropolitan Community College
USA	Miami University
USA	Miami University
USA	Montclair State University
USA	Mount Saint Mary's University
USA	New Mexico State University
USA	Northeastern University
USA	Northern Arizona University
USA	Northern Illinois University
USA	Oberlin College
USA	Ohio State University
USA	Ohio University
USA	Oklahoma State University
USA	Oklahoma State University
USA	Old Dominion University
USA	Pacific University
USA	Pasco Hernando State College
USA	Pensacola State College
USA	Plymouth State University
USA	Portland State University
USA	Quinnipiac University
USA	Radford University
USA	Richmond Hill High School
USA	Shepherd University
USA	Sir Francis Drake High School
USA	Southern Illinois University Edwardsville
USA	St. Philip's College
USA	Stony Brook University
USA	Temple University
USA	Texas Woman's University
USA	The Bush School
USA	The George Washington University
USA	The University of Tampa
USA	UCLA
USA	UM Flint
USA	University at Albany
USA	University at Albany, SUNY
USA	University of Akron
USA	University of Alabama
USA	University of Arizona 
USA	University of Colorado Boulder
USA	University of Colorado Denver
USA	university of florida
USA	University of Idaho
USA	University of Kansas
USA	University of Massachusetts Amherst
USA	University of Massachusetts, Amherst
USA	University of Memphis
USA	University of Minnesota Twin Cities 
USA	University of Mississippi
USA	University of Nebraska–Lincoln
USA	University of North Florida
USA	University of Northern Iowa
USA	University of Pennsylvania
USA	University of Southern California
USA	University of Tennessee at Chattanooga
USA	University of Texas at Tyler
USA	University of Virginia
USA	University of Wisconsin–La Crosse
USA	Valley View Junior High School
USA	Virginia Tech
USA	Washington State University
USA 	University of Houston Downtown 
Venezuela	Executive Coaching Center
```

### PR DESCRIPTION
**Pull request by kguo901**
Fixes #7215

Updated user map data in userMapData.json with new institutions listed in the issue. Additionally, reviewed file for repeated institutions and made consistency fixes in naming.

Ready for review.

### CODE REVIEW

**Comment by kguo901** [comment]
@wkurniawan07 I reverted c668298.

File: src/main/webapp/js/userMapData.json [hunk 1]
```diff
@@ -71,38 +76,47 @@
...
       "IMI - Roosdaal",
+      "KA Avelgem",
       "KdG University College",
       "KU Leuven",
       "Sint Pieters College",
       "Sint-Lievens College",
       "Sint-Lodewijk Campus Van Celst",
       "Sint-Willebrord Heilige Familie",
       "Solvay Brussels School of Economics & Management",
+      "Solvay Brussels School",
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Confirmed the above two are different institutions?

File: src/main/webapp/js/userMapData.json [hunk 2]
```diff
@@ -71,38 +76,47 @@
...
+    "Bermuda": [
+      "Bermuda Institute"
+    ],
     "Botswana": [
       "Alpha",
       "University of Botswana"
     ],
     "Brazil": [
+      "AEDB",
       "Associação Educacional Dom Bosco",
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: These two are the same

File: src/main/webapp/js/userMapData.json [hunk 3]
```diff
@@ -444,28 +485,38 @@
...
       "Universiti Teknologi PETRONAS",
       "Universiti Tunku Abdul Rahman",
+      "Universiti Utara Malaysia",
       "University College of Technology Sarawak",
       "University Kuala Lumpur",
       "University of Malaya",
       "University of Nottingham Malaysia Campus",
-      "University of Reading"
+      "University of Reading",
+      "UTM"
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Same as `Universiti Teknologi Malaysia`

File: src/main/webapp/js/userMapData.json [hunk 4]
```diff
@@ -505,6 +559,7 @@
       "Scala College Alphen aan den Rijn",
       "Sophianum Sgs",
       "Summa College",
+      "Technical University of Eindhoven",
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Same as `TU Eindhoven`

File: src/main/webapp/js/userMapData.json [hunk 5]
```diff
@@ -1331,12 +1446,16 @@
       "Tufts University",
       "UC Berkeley",
       "UC San Diego",
+      "UCLA",
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Same as `University of California Los Angeles`

File: src/main/webapp/js/userMapData.json [hunk 6]
```diff
@@ -1356,13 +1475,17 @@
       "University of Colorado",
       "University of Colorado at Boulder",
       "University of Colorado at Colorado Springs",
+      "University of Colorado Boulder",
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Same as `University of Colorado at Boulder`

File: src/main/webapp/js/userMapData.json [hunk 7]
```diff
@@ -1356,13 +1475,17 @@
...
+      "University of Colorado Denver",
       "University of Dallas",
       "University of Delaware",
       "University of Denver",
+      "university of Florida",
       "University of Florida",
       "University of Georgia",
       "University of Hawaii",
       "University of Houston",
+      "University of Houston Downtown",
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Are these two different institutions?

File: src/main/webapp/js/userMapData.json [hunk 8]
```diff
@@ -1382,14 +1505,17 @@
       "University of Massachusetts",
       "University of Massachusetts Amherst",
       "University of Massachusetts Boston",
+      "University of Massachusetts, Amherst",
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: There's a similar entry two lines above

File: src/main/webapp/js/userMapData.json [hunk 9]
```diff
@@ -1382,14 +1505,17 @@
...
       "University of Memphis",
       "University of Michigan",
       "University of Minnesota",
+      "University of Minnesota Twin Cities",
       "University of Minnesota, Twin Cities",
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: These two are the same :-p

File: src/main/webapp/js/userMapData.json [hunk 10]
```diff
@@ -1382,14 +1505,17 @@
...
       "University of Mississippi",
       "University of Missouri",
       "University of Nebraska",
       "University of Nebraska-Lincoln",
+      "University of Nebraska–Lincoln",
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: And these two. Hello? :-p

Comment (kguo901) [inline src/main/webapp/js/userMapData.json]: Sorry about the obvious repeats. I had removed them in one of my previous commits but forgot to remove them again after I reverted that commit. I will make the changes as soon as possible.

File: src/main/webapp/js/userMapData.json (hunk 1 above)
Comment (kguo901) [inline src/main/webapp/js/userMapData.json]: These refer to the same institution.

File: src/main/webapp/js/userMapData.json (hunk 10 above)
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: No worries @kguo901 Just keep at it and you'll get there :-)

File: src/main/webapp/js/userMapData.json (hunk 1 above)
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: If there are multiple entries referring to the same institution, generally we should keep the longer version (`Solvay Brussels School of Economics & Management` in this case)

Comment (kguo901) [inline src/main/webapp/js/userMapData.json]: Got it. Thank you!

File: src/main/webapp/js/userMapData.json (hunk 7 above)
Comment (kguo901) [inline src/main/webapp/js/userMapData.json]: Yes

File: src/main/webapp/js/userMapData.json [hunk 11]
```diff
@@ -571,6 +624,7 @@
       "Saint Louis University",
       "Saint Mary's University",
       "University of Manila",
+      "University of St. La Salle",
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: I wonder if this is same as `De La Salle University` above

Comment (kguo901) [inline src/main/webapp/js/userMapData.json]: University of St. La Salle and De La Salle University are different universities. University of St. La Salle is located in Bacolod, Philippines whereas De La Salle University is located in Manila, Philippines.

Comment (damithc) [inline src/main/webapp/js/userMapData.json]: I see. Thanks for checking.

**Pull request merged**